src/Sylius/Bundle/CoreBundle/Fixture/Factory/PaymentMethodExampleFactory.php

```diff
@@ -89,6 +93,7 @@ protected function configureOptions(OptionsResolver $resolver): void
             ->setDefault('gatewayName', 'Offline')
             ->setDefault('gatewayFactory', 'offline')
             ->setDefault('gatewayConfig', [])
+            ->setDefault('usePayum', true)
```

Comment:
I think you can add a control on the usePayum type

```suggestion
            ->setDefault('usePayum', true)
            ->setAllowedTypes('usePayum', 'bool')
```